style: increase line-height on announcements page for better readability

navjotmaan opened the pull request:
## Related Issue
Fixes #300 

## Description
The text content on the Announcements page currently has a tight line-height, making it difficult to read, especially for longer updates. Since the content is managed within the main container, I have updated the global typography styles to ensure paragraphs have a more comfortable spacing.

## Changes
- Updated `landing-pages/site/assets/scss/_typography.scss` to set `line-height: 2` for paragraphs within the `.td-main.container`.
- This change specifically targets the main content area to improve legibility without affecting headers or navigation.

choo121600 commented:
The code changes look good to me (LGTM).
However, I personally don’t see an issue with the line spacing.

I think the issue is related to responsiveness.
In the mobile view, some parts are being obscured as shown in the screenshot below, 
and for longer text, it should wrap onto multiple lines but currently doesn’t.

![IMG_F4AAAC1751CD-1](https://github.com/user-attachments/assets/d69d3cef-51f9-4706-870a-fdd9ee4da6a1)

navjotmaan commented:
I have updated the code to address the mobile responsiveness issue. I changed the fixed 306px max-width to 100% and added word-break: break-word along with some padding to ensure the layout is fully fluid on smaller screens. Thanks for the feedback! Please let me know if any further changes are needed.

navjotmaan commented:
Here are the screenshots:
Before: 
<img width="585" height="732" alt="before" src="https://github.com/user-attachments/assets/1690a378-437a-4090-9600-1a756b556779" />

After:
<img width="580" height="753" alt="after" src="https://github.com/user-attachments/assets/2581f312-e133-429c-9550-18ffe59a2c3e" />

choo121600 commented on landing-pages/site/assets/scss/_typography.scss:
```diff
@@ -107,3 +107,12 @@ $bodytext-styles: (
 }
+
+/* Fix for Announcements Page line-height */
+.td-main.container {
```
This CSS selector looks quite broad and seems to affect multiple areas such as the Blog and Home pages.
Additionally, since it is defined with `!important`, it becomes difficult to override the style in more specific contexts.
